Tests/eppo/ConfigurationStoreTests.swift

```diff
@@ -30,6 +33,24 @@ final class ConfigurationStoreTests: XCTestCase {
         )
     }
     
+    override func tearDownWithError() throws {
+        try super.tearDownWithError()
+        // Clean up the configuration file
+        cleanUpConfigurationFile()
+        // Release the reference to ensure it's deallocated
+        configurationStore = nil
+    }
+    
+    private func cleanUpConfigurationFile() {
```

Comment:
nit: does it make sense to add this to `ConfigurationStore`? so the test doesn't need to know where and how configuration is stored